src/ResponsiveProvider/useDebugResponsive.js

```diff
@@ -2,7 +2,11 @@ import { useEffect } from 'react';
 
 const useDebugResponsive = (contextObject, currentMediaType) => {
     useEffect(() => {
-        if (process && process.env && process.env.NODE_ENV === 'development') {
+        if (
+            typeof process !== 'undefined' &&
```

Comment:
I think that replacing this with `process && ...` should be enough, if for some reason a project has `process` as null it will still guarantee that it won't throw an error

Reply:
@SoaresMG I think I'm misunderstanding something from your comment.

`process && ...` is what already exists, and it throws the `Uncaught ReferenceError: process is not defined` error (thus my PR to change it).

If `process` is `null`, the new check will allow this conditional to continue. 

However, when I try to access `process.env`, I get:
`TypeError: can't access property "env", process is null`.

So, perhaps I should change this PR to be:
```js
typeof process !== 'undefined' &&
process !== null &&
process.env &&
process.env.NODE_ENV === 'development'
```

Reply:
Sorry my mistake seems fine for me!

Reply:
@SoaresMG I went ahead and added the `process !== null` check as well.